parts/kubernetesmasteraddons-kube-rescheduler-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rescheduler
+  namespace: kube-system
+  labels:
+    k8s-app: rescheduler
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: rescheduler
+  template:
+    metadata:
+      labels:
+        k8s-app: rescheduler
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      containers:
+      - image: <kubernetesReschedulerSpec>
+        name: rescheduler
+        resources:
+          requests:
+            cpu: <kubernetesReschedulerCPURequests>
+            memory: <kubernetesReschedulerCPULimit>

```

Comment:
This needs to be `<kubernetesReschedulerMemoryRequests>`

Reply:
oops yes thank you!